README.md

```diff
@@ -50,7 +50,7 @@ Supported methods:
 Download and run the dot executable for your OS:
 
 - Windows:
-  - ToDo
+  - Download `dot.zip` from [here](https://drive.google.com/drive/folders/1gR5zIG2RHoMnsaXVzeBlM_Sk1tO4scJx?usp=drive_link), unzip it and then run `dot.exe`

```

Comment:
please use github releases if possible to share zip files. we do the same for the model checkpoints.

Reply:
Do I create a new release for this ?

Reply:
yup. you can then update README and directly commit to master.

Reply:
Do we merge this then? I will update the readme once it's merged and I created the release

Reply:
@Ghassen-Chaabouni i think we decided to use company gdrive for this. please upload the files there and update the url in this PR. not connected to github release now.